lib/models/networkDevice.dart

```diff
@@ -0,0 +1,14 @@
+class NetworkDevice {
+  var iPAddress;
```

Comment:
We should make both of these variables private since they should only be set when the object is created and you already have getter methods